src/js/components/HourPicker.js

```diff
@@ -17,7 +17,7 @@ class HourPicker extends BaseWidget{
     const thisWidget = this;
     // eslint-disable-next-line no-undef
     rangeSlider.create(thisWidget.dom.input);
-    thisWidget.dom.input.addEventListener('input', function(){
+    thisWidget.dom.input.addEventListener('change', function(){
```

Comment:
Mam! Zmień kod hour pickera na ten, który miałaś na samym początku. Wklejam poniżej, jakby Ci się nie chciało przeczesywać historii gita ;)
```
import BaseWidget from '../components/BaseWidget.js';
import {select, settings} from '../settings.js';
import utils from '../utils.js';

class HourPicker extends BaseWidget{
  constructor(wrapper){
    super(wrapper, settings.hours.open);
    const thisWidget = this;

    thisWidget.dom.input = thisWidget.dom.wrapper.querySelector(select.widgets.hourPicker.input);
    thisWidget.dom.output = thisWidget.dom.wrapper.querySelector(select.widgets.hourPicker.output);
    thisWidget.initPlugin();
    thisWidget.value = thisWidget.dom.input.value;
  }

  initPlugin(){
    const thisWidget = this;
    // eslint-disable-next-line no-undef
    rangeSlider.create(thisWidget.dom.input);
    thisWidget.dom.input.addEventListener('input', function(){
      thisWidget.value = thisWidget.dom.input.value;
    });
  }

  parseValue(value){
    return utils.numberToHour(value);
  }

  isValid(){
    return true;
  }

  renderValue(){
    const thisWidget = this;

    thisWidget.dom.output.innerHTML = thisWidget.value;
  }
}

export default HourPicker;
```
Wejdź do pliku BaseWidget.js i usuń linijkę 21 ze sprawdzaniem warunku `!isNaN(newValue) &&`. Nie zauważyłem tego, a to powinno wylecieć już wcześniej, bo nasza wartość jest stringiem nie numberem i dlatego ten warunek nie przejdzie i nowa wartość się nie ustawi ;)
Po tym widget powinien działać dobrze. Przynajmniej u mnie działa ;)